src/builder/module.rs

```diff
@@ -4,9 +4,10 @@ use super::{
     BuildError, Container,
 };
 
+use crate::hugr::hugrmut::sealed::HugrMutInternals;
```

Comment:
nit: I'd drop the reformatting/whitespace/rebracing here and in `src/hugr.rs` so the PR doesn't touch them.